examples/dynamic_selection/sepia-filter-ds/src/sepia_policies.cpp

```diff
@@ -0,0 +1,389 @@
+            uint8_t* dst_image, int i)
+{
+    i *= 3;
+    float temp;
+    temp = (0.393f * src_image[i]) + (0.769f * src_image[i + 1]) + (0.189f * src_image[i + 2]);
+    dst_image[i] = temp > 255 ? 255 : temp;
+    temp = (0.349f * src_image[i]) + (0.686f * src_image[i + 1]) + (0.168f * src_image[i + 2]);
+    dst_image[i + 1] = temp > 255 ? 255 : temp;
+    temp = (0.272f * src_image[i]) + (0.534f * src_image[i + 1]) + (0.131f * src_image[i + 2]);
+    dst_image[i + 2] = temp > 255 ? 255 : temp;
+}
+
+template <typename T>
+int
+invokeDS(int num_offloads, std::vector<sycl::queue>& resources, std::vector<size_t>& npixels,
+         std::vector<sycl::buffer<uint8_t>>& input_buffers, std::vector<sycl::buffer<uint8_t>>& output_buffers)
+{
+
+    T p(resources);
+
+    auto t_begin = std::chrono::high_resolution_clock::now();
+    for (int i = 0; i < num_offloads; ++i)
+    {
+        try
+        {
+            sycl::buffer<uint8_t>& in = input_buffers[i % g_num_images];
+            sycl::buffer<uint8_t>& out = output_buffers[i % g_num_images];
+            size_t num_pixels = npixels[i % g_num_images];
+
+            auto f1 = [&](sycl::queue q, size_t n) {
+                // See what device was actually selected for this queue.
+                cout << "Lambda running on " << q.get_device().get_info<info::device::name>() << "\n";
+
+                // Submit a command group for execution. Returns immediately, not waiting
+                // for command group completion.
+                return q.submit([&](auto& h) {
+                    // This lambda defines a "command group" - a set of commands for the
+                    // device sharing some state and executed in-order - i.e. creation of
+                    // accessors may lead to on-device memory allocation, only after that
+                    // the kernel will be enqueued.
+                    // A command group can contain at most one parallel_for, single_task or
+                    // parallel_for_workgroup construct.
+                    accessor image_acc(in, h, read_only);
+                    accessor image_exp_acc(out, h, write_only);
+
+                    // This is the simplest form sycl::handler::parallel_for -
+                    // - it specifies "flat" 1D ND range(num_pixels), runtime will select
+                    //   local size
+                    // - kernel lambda accepts single sycl::id argument, which has very
+                    //   limited API; see the spec for more complex forms
+                    // the lambda parameter of the parallel_for is the kernel, which
+                    // actually executes on device
+                    h.parallel_for(range<1>(num_pixels), [=](auto i) {
+                        ApplyFilter(image_acc.get_pointer(), image_exp_acc.get_pointer(), i);
+                    });
+                });
+            };
+            ex::submit_and_wait(p, f1, num_pixels);
+        }
+        catch (sycl::exception e)
```

Comment:
```suggestion
        catch (const sycl::exception& e)
```

Reply:
I would prefer not to make many content changes which are not necessary for build & pass in this PR.  Just because the code was moved / formatted doesn't make this PR responsible for the content.
 
That being said, as long as it is only these 2, I can just apply them.